code/modules/martial_arts/combos/superhuman/foot_skewer.dm

```diff
@@ -0,0 +1,16 @@
+/datum/martial_combo/superhuman/foot_skewer
+	name = "Foot Skewer"
+	explaination_text = "We reform our foot into a knife and stab it through our opponent's foot, crippling their mobility."
```

Comment:
```suggestion
	explaination_text = "We reform our foot into a serrated blade and stab it deep into our opponent's foot eviscerating tendons and ligaments, crippling their mobility.
."
```
This is mostly fluff text to make it more epic. Your grammar and spelling was correct.

Reply:
Also my quote is in the wrong spot, heck.

Reply:
Mhm, apologies, I had a major migraine whilst writing my changes.